package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Mor-s-organization/misconfiguration-check#readme",
   "dependencies": {
-    "lodash": "^4.17.20"
+    "axios": "^0.19.2"
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-0155   
**Check Name:** follow-redirects: Exposure of Private Personal Information to an Unauthorized Actor   
**Severity:** HIGH   
**Fixed Version:** 1.14.7   
**Description:** follow-redirects is vulnerable to Exposure of Private Personal Information to an Unauthorized Actor
(This package is used under: axios->follow-redirects)  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-0155